Sources/ChessKit/Piece.swift

```diff
@@ -119,7 +119,7 @@ public struct Piece: Equatable, Hashable {
 
     var graphic: String {
         switch (color, kind) {
-        case (.black, .pawn):   "♟"
+        case (.black, .pawn):   "♟\u{FE0E}"
```

Comment:
Looks great! Only thing is `BoardTests.testPrint()` is failing now and needs to be updated to match this.